tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+###
+### Main entrypoint
+###
+
+- name: Check if Helm binary exists
+  shell: command -v helm >/dev/null 2>&1
+  register: helm_exist
+  ignore_errors: yes
+
+- name: Fail if Helm command does NOT exist
+  fail:
+    msg: "Helm command doesn NOT exist. Please install Helm v{{ helm_version }}"
+  when: helm_exist.rc != 0
+
+- name: Get Helm verson
+  command: "helm version --client --template '{{ '{%' }} raw %}{{ '{{' }} .Client.SemVer }}{{ '{%' }} endraw %}'"
+  register: helm_local_version
```

Comment:
suggestion: prefixing vars registered by the role with underscore, as mentioned above.

Reply:
Done. Thx